js/main.js

```diff
@@ -36,12 +45,12 @@ document.addEventListener('DOMContentLoaded', function () {
 });
 
 function renderProfile() {
-  var h1 = document.createElement('h1');
+  h1 = document.createElement('h1');
   h1.textContent = data.profile.fullName;
   $views.viewProfile.appendChild(h1);
 
-  var row = document.createElement('div');
-  row.setAttribute('class', 'row');
+  row = document.createElement('div');
```

Comment:
This pattern of defining variables globally for later is a bad practice ,if you hard code the html itll get easier to work with.
Not sure what the row is used for though, maybe you can explain it to me